extensions/amp-story/1.0/amp-story-open-page-attachment.js

```diff
@@ -19,10 +19,13 @@
  */
 import {AttachmentTheme} from './amp-story-page-attachment';
 import {LocalizedStringId} from '#service/localization/strings';
-import {Services} from '#service';
 import {computedStyle, setImportantStyles} from '#core/dom/style';
 import {getLocalizationService} from './amp-story-localization-service';
-import {getRGBFromCssColorValue, getTextColorForRGB} from './utils';
+import {
```

Comment:
On this file line 277, we want to pass directly `pageEl.getAmpDoc()` (or however we want to grab the AmpDoc), but not the element itself.